src/System.Runtime/tests/System/StringTests.netcoreapp.cs

```diff
@@ -33,6 +33,25 @@ public static void Ctor_CharSpan(char[] valueArray, int startIndex, int length,
         }
 
         [Theory]
+        [InlineData("")]
+        [InlineData("a")]
+        [InlineData("this is a test")]
+        [InlineData("\0\u8001\u8002\ufffd\u1234\ud800\udfff")]
+        public static void Create_ReturnsExpectedString(string expected)
+        {
+            char[] input = expected.ToCharArray();
+            string result = string.Create(input.Length, input, (span, state) =>
```

Comment:
What's supposed to happen when `action` is null?

Reply:
It should throw.  I think in my tired stupor I subconsciously saw "span" and thought "that can't be null" :)